apps/dashboard/app/javascript/files/data_table.js

```diff
@@ -16,11 +16,14 @@ let table = null;
 jQuery(function () {
     table = new DataTable();
 
-
     /* END BUTTON ACTIONS */
 
     /* TABLE ACTIONS */
     
+    window.onpopstate = function(event) {
+      table.reloadTable(location.href)
```

Comment:
I think we need to use `goto` here instead - `reloadTable` doesn't update the breadcrumbs.

Reply:
🤦 Good catch, thanks.

Reply:
Although I'm confused. There is no `goto` in JS or JQuery?

Reply:
That `table` object has a `goto` method:

https://github.com/OSC/ondemand/blob/cdfbab078b7b92efcf9c1aadc75fda7351581f6e/apps/dashboard/app/javascript/files/data_table.js#L368